model/task_lifecycle.go

```diff
@@ -817,6 +818,16 @@ func UpdateUnblockedDependencies(t *task.Task) error {
 		if err := UpdateUnblockedDependencies(&blockedTask); err != nil {
 			return errors.WithStack(err)
 		}
+
+		buildsToUpdate[blockedTask.BuildId] = true
+	}
+
+	var buildIds []string
```

Comment:
It makes my eye twitch slightly that it's `buildID` above but `buildId` here, but I can get over it.

Reply:
haha i didnt even realize that you wrote the same logic up there 
maybe we can try to put it in the style guide so we all write it in the same way! 
my preference is Id because all capital makes me feel like it should stand for something

Reply:
ahh ill update it to match it then!